modules/components/src/NestedTreeView/NestedTreeView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactTreeView from './ReactTreeView';
+import './style.css';
+
+const NestedTreeView = ({ dataSource, depth = 0 }) =>
+  dataSource.map(
+    ({ title, id, children, isHeader }, i) =>
+      children ? (
+        <ReactTreeView
+          key={id || i}
```

Comment:
Instead of `i` could you use `title` or ``${title}-${i}``. will avoid weird bugs down the line :)

Reply:
definitely!